doc/Contributions.md

```diff
@@ -0,0 +1,63 @@
+Our contributions to the ecosystem.
+
+## Our contributions to third-party opensource projects
+
+### XMPPframework for iOS
+
+https://github.com/robbiehanson/XMPPFramework
+
+* XEP-0363: HTTP File Upload: https://github.com/robbiehanson/XMPPFramework/pull/730
+* XEP-0313: Message Archive Management: https://github.com/robbiehanson/XMPPFramework/pull/733
+* XEP-0030: Service Discovery: https://github.com/robbiehanson/XMPPFramework/pull/736
+* MUC light: https://github.com/robbiehanson/XMPPFramework/pull/750
+* Token-based reconnection: https://github.com/robbiehanson/XMPPFramework/pull/758
+* Revamped README: making people feel like this is a well mantained and up to date framework
+* Created a way to Mock a piece of the framework to improve the way we write tests
+
+### Smack for Android
+
+https://github.com/igniterealtime/Smack
```

Comment:
Probably it the following would be enough:
`[igniterealtime/Smack](https://github.com/igniterealtime/Smack)`